### DISCUSSION TITLE
Coordinator turbine and boiler off-line

### DISCUSSION THREAD

**Discussion by ghost** [MikaylaFischler/cc-mek-scada#304] [Support] [answered]
I set up one RTU for 8 turbines and 4 boilers. The RTU says everything's fine and online but on the coordinator monitor the turbines and boilers for reactor 2-4 are all off-line. Did I do something wrong? I did try to setup everything again but same thing is happening.
![RTU](https://github.com/MikaylaFischler/cc-mek-scada/assets/75527884/c60231c1-7d24-415c-bc6b-755d1b87d0d9)
![coordinator](https://github.com/MikaylaFischler/cc-mek-scada/assets/75527884/f7f0aa2b-1640-4b72-aeae-ecfd2b23aaca)

**Comment by MikaylaFischler** [comment, marked as answer]
"Index" is specific to that unit. So, it should say 

"TURBINE 1 -> UNIT 1"
"TURBINE 2 -> UNIT 1"
"TURBINE 1 -> UNIT 2"
"TURBINE 2 -> UNIT 2"
etc, same for boiler
"BOILER 1 -> UNIT 1"
"BOILER 1 -> UNIT 2"
etc

**Reply by ghost** [reply]
Everything is working now, thank you very much!
